research/src/pages/button.mdx

```diff
@@ -16,6 +16,20 @@ import ComponentCoverage from '../components/component-coverage'
 
 <Anatomy component="Button" />
 
+## Accessibility
+
+- Use a touch target area of `44px` by `44px`
```

Comment:
This one needs to be refined, questioned, or removed.

Nearly every button on the web is smaller than 44x44. Where did this guidance come from? Is it correct?

For example, github buttons are 34px tall.

Fabric buttons I believe are 32.

Reply:
Also, since these are the "Open UI" specs, we need to document what is existent in the broader open community.  This will have to be validated with some sources.  The specs here should descriptively document what exists in the community already opposed to prescribe something.

Think of Wikipedia, which doesn't allow original content but instead links to sources.  In a similar way, phase 1 of Open UI needs to link to sources and aggregate them.  We should not prescribe things we want.

If Microsoft has a certain accessibility requirement then it should live here but we need to introduce a way of noting that this is Microsoft's requirement.  Think of how an MDN page shows APIs and properties that exist in HTML/CSS/JS specs, but also allows vendor specific APIs and properties to be surfaced.  However, it is clearly noted that these are vendor specific variants of a thing, opposed to presenting it as if it were the spec itself.

This capability of the Open UI doc build does not yet exist.  So, we need to figure how we want to display this kind of information, accessibility information that comes from specific sources.